apps/backend/src/connect.js

```diff
@@ -0,0 +1,30 @@
+const { MongoClient } = require("mongodb");
+// Replace the following with your Atlas connection string                                                                                                                                        
+const url = "mongodb+srv://helping_hand:X26igJL9@clusterhelpinghand.c1myh2s.mongodb.net/?retryWrites=true&w=majority";
```

Comment:
I think you aren't using this file from what I can see, if that is the case, maybe delete the file. If it is needed, then please remove the db url